RecoMuon/MuonIdentification/plugins/MuonRefProducer.h

```diff
@@ -21,6 +21,7 @@
 #include "FWCore/Framework/interface/global/EDProducer.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "DataFormats/MuonReco/interface/Muon.h"
+#include "DataFormats/MuonReco/interface/MuonFwd.h"
```

Comment:
Muon.h and MuonFwd.h are repeated in the MuonRefProducer.cc
Would you like to clean up the MuonRefProducer.cc implementation file before merging this PR?